content/modules/ROOT/pages/module-00.adoc

```diff
@@ -212,36 +260,43 @@ Add Build Ansible Lightspeed Job Template:
 
 image::add_lightspeed_jt_creator.png[Add Lightspeed JT Creator Step]
 
-Click Next, then Finish.
+[start=21]
+22. Click Next, then Finish.
 
 image::blue_next_button.png[]
 image::blue_finish_button.png[]
 
-Final workflow visual:
+[start=23]
+23. Final workflow visual:
 
 image::workflow_final_prompt_generation.png[Final Workflow]
 
-Click Save to finalize.
+[start=24]
+24. Click Save to finalize.
 
 image::save_button.png[]
 
 == Trigger the Workflow
 
-Run the `❌ Break Apache` job template. This inserts an invalid directive in Apache config and restarts the service.
+[start=25]
+25. Run the `❌ Break Apache` job template. This inserts an invalid directive in Apache config and restarts the service.
 
 image::run_break_apache.png[]
 
-Go to Automation Decisions(Event-Driven Ansible) → Rulebook Activations. Confirm EDA(Event-Driven Ansible) picked up the event.
+[start=26]
+26. Go to Automation Decisions(Event-Driven Ansible) → Rulebook Activations. Confirm EDA(Event-Driven Ansible) picked up the event.
 
 image::eda_trigger_capture.png[]
 
-Go to Automation Controller → Jobs. Confirm workflow execution.
+[start=27]
+27. Go to Automation Controller → Jobs. Confirm workflow execution.
 
 image::workflow_triggered_jobs.png[]
 
 When the workflow completes you will see a green ✅ Success
```

Comment:
I recommend just adding this to step 27. It looks strange just hanging out after the screenshot by itself.